Library/Formula/swatchbooker.rb

```diff
@@ -5,9 +5,9 @@ class Swatchbooker < Formula
   sha256 "c0c8bf038156337f1eebdf6f7c99f5b7a8e8f9a332cd625c57269ddc8ba18eb7"
 
   depends_on :python
-  depends_on "pillow" => [:python, "PIL"]
   depends_on "little-cms" => "with-python"
   depends_on "pyqt"
+  depends_on "homebrew/python/pillow"
```

Comment:
It should be fine to vendor. We [do so](https://github.com/Homebrew/homebrew/blob/master/Library/Formula/mitmproxy.rb#L93-L96) in `mitmproxy` without any obviously horrific problems. Possible the instruction to use `pip` in this formula previously is a relatively old relic.